src/scripts/statusbar.ts

```diff
@@ -28,6 +31,13 @@ namespace Scripts {
             Shared.AddArray(GlobalEnum.customStatusBars, statusBars);
 
             const gump = Orion.CreateCustomGump(parseInt(serial, 16));
+
+            if (coordinates) {
+                gump.SetX(coordinates.x);
+                gump.SetY(coordinates.y);
+                gump.Update();
+            }
```

Comment:
tu ma napadla sranda.. možno kravina.. Nebolo by fajn, keby to offsetlo Y o `Y * numberOfStatusbars`? tzn začneš napr na x = 30, y = 30, statusbar má výšku napr 50.. tak prvý statusbar ti dá na 30.30, druhý na 30.85 napr (s 5pt gap), ...

len nápad.. aby si to nemusel rovnať

Reply:
to si resis ve scriptu ktery tento create vola a ne zde, kde chces umistit primo na pozici kterou sis vybral.
Takze ne